build.gradle

```diff
@@ -1,6 +1,9 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext{
+        supportBuildVersion = '27.1.0'
```

Comment:
Use 27.1.1 because the version 27.1.0 has some bugs that are solved in 27.1.1
I faced that bugs and they are fixed in 27.1.1